var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -50,9 +51,16 @@ class Neuron(Package):
     depends_on('libtool',    type='build')
     depends_on('pkgconfig',  type='build')
 
+    # Readline became incompatible with Mac so we use neuron internal readline.
+    # HOWEVER, with the internal version there is a bug which makes Vector.as_numpy() not work!
+    if platform.system() != "Darwin":
```

Comment:
most of the packages in spack use `sys.platform` not sure because `platform.system` is heavy :) was just reading here: https://stackoverflow.com/questions/4553129/when-to-use-os-name-sys-platform-or-platform-system.

we could use when clause for one liner : `depends_on('readline',  when=sys.platform != 'darwin')`

Reply:
The 1-liner is nice, I didn't know we could use something not the spec.
By what I read `platform.system` is a bit more reliable, for for this case doesn't matter much.

Reply:
https://stackoverflow.com/questions/1854/python-what-os-am-i-running-on/15674751
> Slightly more consistent output. i.e. platform.system() returns "Windows" instead of "win32". 
> sys.platform also contains "linux2" on old versions of Python while it contains just "linux" on newer ones. platform.system() has always returned just "Linux"

Reply:
ah ok!